app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -64,16 +64,33 @@
             app:layout_constraintEnd_toEndOf="@+id/add_profile_activity_user_image_view"
             app:layout_constraintRight_toRightOf="@+id/add_profile_activity_user_image_view" />
 
-          <TextView
-            android:id="@+id/add_profile_activity_required_heading_text_view"
-            android:layout_width="match_parent"
+          <LinearLayout
+            android:id="@+id/add_profile_activity_required_relative_layout"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginTop="32dp"
-            android:contentDescription="@string/required_fields_content_description"
-            android:text="@string/add_profile_required"
-            android:textColor="@color/oppiaPrimaryText"
-            android:textSize="12sp"
-            app:layout_constraintTop_toBottomOf="@+id/add_profile_activity_user_image_view" />
+            android:orientation="horizontal"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/add_profile_activity_user_image_view">
+
+            <TextView
+              android:id="@+id/add_profile_activity_required_heading_star"
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content"
+              android:contentDescription="@string/required_fields_content_description_star"
+              android:text="@string/add_profile_required_star"
+              android:textColor="@color/oppiaPrimaryText"
+              android:textSize="12sp" />
+
+            <TextView
```

Comment:
@rishidyno I was just trying out another solution where we can actually solve this without introducing another text view.

Simply in earlier TextView change string from
`android:text="@string/add_profile_required"`
to 
`android:text="@{'*' + @string/add_profile_required}"`

And it should work correctly.

Reply:
I have made the required changes, it work fine on devices but does not show on my Design tab